src/main/java/uk/gov/hmcts/cmc/claimstore/services/DefendantResponseService.java

```diff
@@ -78,8 +79,7 @@ public Claim save(
         eventProducer.createDefendantResponseEvent(claimAfterSavingResponse);
         ccdEventProducer.createCCDDefendantResponseEvent(claimAfterSavingResponse, authorization);
 
-        appInsights.trackEvent(getAppInsightsEventName(response),
```

Comment:
@tharacka please remove the implementation of, as not used anymore inside:`uk.gov.hmcts.cmc.claimstore.services.DefendantResponseService#getAppInsightsEventName`